src/main/scala/fastq.scala

```diff
@@ -269,7 +269,7 @@ case object fastq {
 
     def parseFastqPhred33: Iterator[Option[FASTQ]] =
       lines
-        .grouped(4)
+        .grouped(4).filter(_.size == 4) // grouped is broken
```

Comment:
> And it is broken, check http://stackoverflow.com/a/7959133/614394 and http://www.scala-lang.org/old/node/7939

Sometimes, reading the [documentation](http://scala-lang.org/api/2.11.8/index.html#scala.collection.Iterator@grouped[B>:A](size:Int):Iterator.this.GroupedIterator[B]) is more useful than reading old discussions about somebody's wrong decisions.

> it does not have the right behavior

Care to elaborate? The only difference I see is the exact match on `"+"` vs. `startsWith`, which is easy to change.